### PR TITLE
[#896] Undo EFSL changes

### PR DESCRIPTION
**Pull request by mkarg**
Closing #896 

Undoing recent EFSL changes (originally requested by @kwsutter) as requested by @kwsutter.

### CODE REVIEW

**Comment by mkarg** [comment]
@kwsutter Is *this* what you actually requested with your recent posting?

@ivargrimstad Is this what the EF wants to have it?

@chkal As this is a simple reverse of your original commit, do you like to add a commit ontop which keeps some unrelated changes (like removing the Oracle copyright)?

File: jaxrs-api/pom.xml [hunk 1]
```diff
@@ -205,9 +205,11 @@
                         <configuration combine.self="override">
                             <doctitle>${apidocs.title}</doctitle>
                             <bottom>
-                                <![CDATA[<p align="left">Copyright &#169; 2018, 2020 Eclipse Foundation.<br>Use is subject to <a href="{@docRoot}/resources/EFSL.html" target="_top">license terms</a>.]]>
+                                <![CDATA[Copyright &#169; 1996-2017,
```
Comment (hussainnm) [inline jaxrs-api/pom.xml:208]: The pom.xml changes need not be reverted.

File: jaxrs-api/src/main/javadoc/resources/EFSL.html [hunk 2]
```diff
@@ -20,9 +20,8 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
     <li>All existing copyright notices, or if one does not exist, a notice
         (hypertext is preferred, but a textual representation is permitted)
         of the form: &quot;Copyright &copy; [$date-of-document]
-        Eclipse Foundation, Inc.
-        <a href="https://www.eclipse.org/legal/efsl.php">
-            https://www.eclipse.org/legal/efsl.php</a>&quot;
+        &ldquo;Eclipse Foundation, Inc. &lt;&lt;url to this license&gt;&gt;
```
Comment (hussainnm) [inline jaxrs-api/src/main/javadoc/resources/EFSL.html:23]:
```suggestion
        Eclipse Foundation, Inc. &lt;&lt;url to this license&gt;&gt;
```
Remove &ldquo;

File: jaxrs-api/src/main/javadoc/resources/EFSL.html [hunk 3]
```diff
@@ -43,10 +42,9 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
 
 <p>The notice is:</p>
 
-<p>&quot;Copyright &copy; 2018, 2020 Eclipse Foundation. This software or
-    document includes material copied from or derived from
-    Jakarta RESTful Web Services <a href="https://jakarta.ee/specifications/restful-ws/3.0/">
-    https://jakarta.ee/specifications/restful-ws/3.0/</a>.&quot;</p>
+<p>&quot;Copyright &copy; 2018 Eclipse Foundation. This software or
```
Comment (hussainnm) [inline jaxrs-api/src/main/javadoc/resources/EFSL.html:45]:
```suggestion
<p>&quot;Copyright &copy; [$date-of-document] Eclipse Foundation. This software or
```

File: jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc [hunk 4]
```diff
@@ -23,10 +23,9 @@ endif::[]
 
 Release: {revdate}
 ....
-== Copyright
+=== Copyright
```
Comment (hussainnm) [inline jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc:26]: Keep existing changes

File: jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc [hunk 5]
```diff
@@ -23,10 +23,9 @@ endif::[]
...
 
-Copyright (c) 2018, 2020 Eclipse Foundation.
-https://www.eclipse.org/legal/efsl.php[]
+Copyright (c) 2019 Eclipse Foundation.
```
Comment (hussainnm) [inline jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc:28]: Keep existing changes

File: jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc [hunk 6]
```diff
@@ -64,9 +63,9 @@ specification is expressly prohibited.
 
 The notice is:
 
-"Copyright (c) 2018, 2020 Eclipse Foundation. This software or
-document includes material copied from or derived from Jakarta(R) RESTful
-Web Services https://jakarta.ee/specifications/restful-ws/3.0/[]."
+"Copyright (c) 2018 Eclipse Foundation. This software or
```
Comment (hussainnm) [inline jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc:66]:
```suggestion
"Copyright (c) [$date-of-document] Eclipse Foundation. This software or
```

**Comment by mkarg** [comment]
> I have commented in few places where the existing changes are correct and a few corrections.
> The final license text output should match exactly as https://www.eclipse.org/legal/efsl.php

Thanks. Can you please send a PR agains my branch with those changes? I will be happy to merge it into this PR.

What do others think? Agreed with @hussainnm's claims?

**Comment by hussainnm** [comment]
@mkarg This branch is created directly under this project and not in your fork. If I create a PR it will create a new one here, can you cherry pick my commit from my fork?

https://github.com/hussainnm/jaxrs-api/tree/mkarg/undo-efsl-changes-896

**Comment by mkarg** [comment]
> @mkarg This branch is created directly under this project and not in your fork. If I create a PR it will create a new one here, can you cherry pick my commit from my fork?
> 
> https://github.com/hussainnm/jaxrs-api/tree/mkarg/undo-efsl-changes-896

Sure. Thanks! :-)

**Comment by hussainnm** [comment]
> To be honest, I'm quite confused now. We must really keep these references like:
> 
> ```
> [$date-of-document]
> ```
> 
> ?? If so, +1 for merging this. But IMO this is really weird.

In Kevin's email he has mentioned this point (see quoted text below). From the document standpoint the Copyright notice already has the correct years in the Copyright section above the license text.

> It has been explained to me that the EFSL text must stay as it was originally written (even with the cryptic references to other URLs or Specifications).

**Comment by mkarg** [comment]
> LGTM
> I think the "cryptic stuff" in the adoc are placeholders and will be updated somehow somewhere, but I am not sure.
> Regarding the html-version, I have no idea if that happens there as well. But as long as it looks the same, that should be good enough.

Spoken frankly, the PMC's communication still is a mess. If they want things to get changed in a project, why can't they open an issue in the project and clearly write the **exact** wanted change there. Or even better, as Kevin actually is a committer to JAX-RS, why can't he simply post a PR? So nobody needs to interprete these vague directions.

**Comment by mkarg** [comment]
@kwsutter Please review ASAP. Thanks.

**Comment by spericas** [comment]
@kwsutter Could you review this?

**Comment by spericas** [comment]
@mkarg It's a fairly minor change after all, can we just go ahead and merge this without additional approvals?

**Comment by mkarg** [comment]
> @mkarg It's a fairly minor change after all, can we just go ahead and merge this without additional approvals?

According to our committer rules we need to wait four more days, as I had not previously announced it as fast-track. Also, the change was originally requested by the EF via @kwsutter and I doubt we (the committers) have an understanding of the implications if we do it wrong. So I propose that Kevin or anybody else of the PMC, WG or EF *immediately* confirms that we fixed it correctly before me merge it *or* we merge after nobody of those stakeholders object until then. IANAL and I don't want to fix it *again* just because of a forgotten comma somewhere.

**Comment by mkarg** [comment]
Just noticed that Ivar approved the changes. I don't know his actual position in the EF, but for me, this is "official enough". So if nobody objects to our project lead's plea, I hereby prematurely terminate the waiting time (despite our committer rules) and **immediately** merge  this.

@spericas Are you OK with this?

**Comment by spericas** [comment]
> Just noticed that Ivar approved the changes. I don't know his actual position in the EF, but for me, this is "official enough". So if nobody objects to our project lead's plea, I hereby prematurely terminate the waiting time (despite our committer rules) and **immediately** merge this.
> 
> @spericas Are you OK with this?

Yes, thx.

**Comment by mkarg** [comment]
Done. @spericas HTH. :-)

**Pull request merged**